server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -71,8 +71,8 @@ const (
 )
 
 var (
-	chunkFileSizeBytes     = flag.Int("storage.chunk_file_size_bytes", 0, "How many bytes to buffer in memory before flushing a chunk of build protocol data to disk.")
-	enableChunkedEventLogs = flag.Bool("storage.enable_chunked_event_logs", false, "If true, Event logs will be stored separately from the invocation proto in chunks.")
+	chunkFileSizeBytes     = flag.Int("storage.chunk_file_size_bytes", 3_000_000 /* 3 MB */, "How many bytes to buffer in memory before flushing a chunk of build protocol data to disk.")
+	enableChunkedEventLogs = flag.Bool("storage.enable_chunked_event_logs", true, "If true, Event logs will be stored separately from the invocation proto in chunks.")
```

Comment:
I recently (May 7th) had a customer email me about some issues when enabling this flag on Amazon using S3. 

We might want to test that out before enabling it for everyone (can split this out so it doesn't block the rest of these changes).

Reply:
Fixed!